src/main/java/com/authservice/authservice/exception/JWTExceptionHandler.java

```diff
@@ -0,0 +1,18 @@
+package com.authservice.authservice.exception;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+@Slf4j
+public class JWTExceptionHandler {
+
+    @ExceptionHandler(value = NotFoundException.class)
+    public ResponseEntity<String> userNotFoundException(NotFoundException exception) {
```

Comment:
name should be `notFoundException`

Reply:
Done.

Reply:
See my class name is already NotFoundException.class i just created a function with userNotFoundException as i changed that function to notFoundException.

Reply:
In UserDetailsServiceImpl.loadUserByUserName function is throwing it's own exception UserNameNotFoundException = it is a spring exception